### DISCUSSION TITLE
:pray: We need an early user to test Seia!

### DISCUSSION THREAD

**Discussion by abiriadev** [abiriadev/seia#1] [Announcements]
Seia is still in its early stages, and we need some users to test it out.

If you're interested, please follow the below checklist:

```md
- [ ] Npm download works.
- [ ] Sample project works fine.
- [ ] Can use router library wihtout any issues.
- [ ] Can use external components without any issues.
- [ ] Build success.
- [ ] Running SSR server works.
```

You can copy-paste the above checklist to your comment, and mark the items as you go through them.

If you encounter any issues, while following these steps, please let us know by creating an issue!

We appreciate your help!

**Comment by kangjun-lee** [comment]
- [x] Npm download works.
- [x] Sample project works fine.
- [ ] Can use router library wihtout any issues.
  - I tried to use ReactRouter, and it didn't work
  - I think there should be a guide on how to use router library.
- [x] Can use external components without any issues.
  - I've tested the mantine library
  - I was able to render the component with 'use client' without problems
- [x] Build success.
- [x] Running SSR server works.
- [x] Yurizono Seia is kawaii

**Reply by abiriadev** [reply]
@gangjun06 Thanks for feedback!

> * I tried to use ReactRouter, and it didn't work

Have you tried [this guide](https://reactrouter.com/en/main/guides/ssr) when setting up react router dom?

> * I think there should be a guide on how to use router library.

Definitely, it should. I will add more guides to the README later.
